File: Demo/app/src/main/java/com/example/demo/MainActivity.java
package com.example.demo;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    Button btnSignIn;

    private static final String SHARED_PREFS_NAME = "MY_SHARED_PREF";


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        UserInfo.instance.setContext(this);

        btnSignIn = (Button)findViewById(R.id.btnSignIn);

        btnSignIn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent signIn = new Intent(MainActivity.this, SignIn.class);
                startActivity(signIn);
            }
        });
    }
}

File: OrderManagementApplication/app/src/main/java/com/ngsown/ordermanagementapplication/MainActivity.kt
package com.ngsown.ordermanagementapplication

import android.app.ProgressDialog
import android.content.Context
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import com.google.firebase.database.DatabaseReference
import com.google.firebase.database.ValueEventListener
import com.google.firebase.database.DataSnapshot
import com.google.firebase.database.DatabaseError
import com.google.firebase.database.ktx.database
import com.google.firebase.ktx.Firebase

data class LoginInfo(val username: String, val password: String){}

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var txtLogo = findViewById<TextView>(R.id.txtLogo)
        txtLogo.text = "Smart Food Court System"
        var btnLogo = findViewById<Button>(R.id.btnLogo)
        btnLogo.setOnClickListener{
            loadLoginActivity()
        }
    }
    private fun loadLoginActivity(){
        val intent: Intent = Intent(this, LoginActivity::class.java)
        startActivity(intent)
    }
}